changelogs/fragments/6572-nmcli-add-support-loopback-type.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - nmcli - add support for new connection type: ``loopback``(https://github.com/ansible-collections/community.general/issues/6572).
```

Comment:
Either remove the `:`, or add quotes around the entry. Otherwise the YAML parser things this is a dictionary with key "nmcli - add support for new connection type" :)

Reply:
Already resolved, GH hid this comment after refreshing so I couldn't delete it anymore before sending :)